README.md

```diff
@@ -1,20 +1,14 @@
-# TheMovieDB App Part One
+# TheMovieDB App Part Three
```

Comment:
Good idea to update the `README.md`!! 🚀  Buuut I wouldn't delete the previous "Parts". Create new sections in the `README` with the new tasks, but without deleting the previous work :D

Reply:
Done.